### ISSUE TITLE
<<print $var >> not working

### ISSUE THREAD

**Issue by eporral1** [InfiniteAmmoInc/Yarn#39]
Is it possible to print the content of a variable ?

**Comment by charblar** [comment]
Not at this time, this may be possible in the near future when we port to electron and implement [bondage.js](https://github.com/jhayley/bondage.js) however.

**Comment by blurymind** [comment]
Yes, I can potentially add this and wrote some code for bondage.js to be able retrieve the contents of <<>> as a string, while the story is parsed.
This has not been merged yet, but if it does- It will enable me to add it to yarn's electron port :)
